internal/generate/testdata/clusterserviceversions/output/with-ui-metadata.clusterserviceversion.yaml

```diff
@@ -253,3 +253,15 @@ spec:
     targetPort: 9443
     type: MutatingAdmissionWebhook
     webhookPath: /mutate-cache-my-domain-v1alpha1-memcached
+  - admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    conversionCRDs:
+      - memcacheds.cache.example.com
+    deploymentName: memcached-operator-controller-manager
+    generateName: ""
```

Comment:
There is no way for a user to set the `generateName` in WebhookClientConfig in CRD. Is it fine, if it appears "", or can it be manually set? @estroz

Reply:
You can set this manually with something indicative like:
```go
sort.Strings(crdNames)
joinedResourceNames := strings.Builder{}
for _, name := range crdNames {
	if name != "" {
		joinedResourceNames.WriteString(strings.Split(name, ".")[0])
	}
}
whDesc.GenerateName = fmt.Sprintf("c%s.kb.io", joinedResourceNames.String())
```